nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ValidateRecord.java

```diff
@@ -180,6 +180,19 @@
         .defaultValue("true")
         .required(true)
         .build();
+    static final PropertyDescriptor COERCE_TYPES = new PropertyDescriptor.Builder()
+            .name("coerce-types")
+            .displayName("Force Types From Schema")
+            .description("If enabled, the processor will coerce every field to the type specified in the schema. "
```

Comment:
I would mention in the description (and maybe also in the property name) that the schema here is the Record Reader's schema, not the validation schema (which may be a different one).

Reply:
Thank you! I pushed a new commit with the clarification.